frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx

```diff
@@ -55,40 +55,58 @@ function RecordingDuration({
     )
 }
 
-function ActivityIndicators({
-    recording,
-    onPropertyClick,
-    iconClassnames,
-}: {
-    recording: SessionRecordingType
-    onPropertyClick?: (property: string, value?: string) => void
-    iconClassnames: string
-}): JSX.Element {
-    const { recordingPropertiesById, recordingPropertiesLoading } = useValues(sessionRecordingsListPropertiesLogic)
+interface GatheredProperty {
+    property: string
+    value: string | undefined
+    tooltipValue: string
+}
 
-    const recordingProperties = recordingPropertiesById[recording.id]
-    const loading = !recordingProperties && recordingPropertiesLoading
+const browserIconPropertyKeys = ['$geoip_country_code', '$browser', '$device_type', '$os']
+const mobileIconPropertyKeys = ['$geoip_country_code', '$app_name', '$device_type', '$os_name']
```

Comment:
I would say lets not display app name - we should have as few, useful properties as possible here